extensions/amp-story/amp-story.md

```diff
@@ -90,12 +90,10 @@ The following markup is a decent starting point or boilerplate. Copy this and sa
 <head>
     <meta charset="utf-8">
     <script async src="https://cdn.ampproject.org/v0.js"></script>
-    <script async custom-element="amp-story"
-            src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
     <title>Hello, amp-story</title>
     <link rel="canonical" href="http://example.ampproject.org/my-story.html" />
-    <meta name="viewport"
-          content="width=device-width,minimum-scale=1,initial-scale=1">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
```

Comment:
@newmuis Looks much neater without the whitespace

Reply:
Sorry, I mean the whitespace in other places in the file 😃  It looks like you have changed the indentation of many lines below.

Thanks for all the changes!

Reply:
I see I see... Let me sort it out

Reply:
@newmuis  when you have a chance, kindly review the commit! :) Thanks

Reply:
Sorry for all the confusion ☹️ 

What I'm trying to say is that this change should really only consist of line 100:

```html
<amp-story standalone title="Hello Story" publisher="The AMP Team" publisher-logo-src="https://example.com/logo/1x1.png" poster-portrait-src="https://example.com/my-story/poster/3x4.jpg">
```

All the other changes in this file should be reverted

Reply:
It's okay @newmuis this is clear. Let me do so

Reply:
Hello @newmuis for the life of me, I can't find a way to revert the changes I had made, can you help. I'd really like to close the issues. Thank you

Reply:
Hello @newmuis Did you get a chance to look at my struggles?

Reply:
It's easy, alright I have changed it.